labelbox/data/annotation_types/geometry/rectangle.py

```diff
@@ -26,18 +28,27 @@ def geometry(self) -> geojson.geometry.Geometry:
             [self.start.x, self.start.y],
         ]])
 
-    def raster(self, height: int, width: int,
-               color=(255, 255, 255)) -> np.ndarray:
+    def raster(self,
+               height: Optional[int] = None,
+               width: Optional[int] = None,
+               color=(255, 255, 255),
+               thickness=-1,
+               canvas=None) -> np.ndarray:
         """
         Draw the rectangle onto a 3d mask
-
         Args:
             height (int): height of the mask
             width (int): width of the mask
             color (int): color for the rectangle. Only a single int since this is a grayscale mask.
         Returns:
             numpy array representing the mask with the rectangle drawn on it.
         """
-        canvas = np.zeros((height, width, 3), dtype=np.uint8)
+        if canvas is None:
+            if height is None or width is None:
+                raise ValueError(
+                    "Must either provide canvas or height and width")
+            canvas = np.zeros((height, width, 3), dtype=np.uint8)
```

Comment:
Could we have a helper function for that in Geometry? (I guess `raster` is an abstract method in Geometry?)